### ISSUE TITLE
Expose member anchors

### ISSUE THREAD

**Issue by wfdd** [everypolitician/viewer-sinatra#10135]
I'm thinking there should be a way to link to specific politicians.  We could show a little clickable anchor icon when hovering over a person card, though it looks like the anchors themselves are hex UUIDs, which aren't very berry user-friendly and also probably transient.  The facet switcher uses anchors for state, so we'd have to figure out a way to get the two to play nicely together.  Design-wise, the focussed person card could have its background painted a different colour to draw attention to it.

Basically, I'm pondering how to link to MPs on EveryPolitician from [openpatata/question-viewer](https://github.com/openpatata/question-viewer).  EveryPolitician has got a pretty solid presentation of MPs I'd be averse to replicate.

**Comment by tmtmtmtm** [comment]
Will ponder the wider question, but as a quick note, the `id` on the `person-card` div uses the underlying UUID from the EveryPolitician data, which should be stable.

A URL of the form http://everypolitician.org/cyprus/house-of-representatives/term-table/2011.html#search=2bccfeee-aeb1-4223-b0b9-2160ff3625a2 will already "work" to the extent that it will restrict the page to just that person, but someone arriving on that page without already knowing what's going on is almost certainly going to find it very confusing.

**Comment by wfdd** [comment]
Ah I hadn't thought to search for the UUID.  It would help a little if it were to jump to the content (or the facet switcher) and perhaps after #9772 I could use their Greek names.
